help: Document new UI for testing mobile push notifications. (#28076)
zulip/zulip: help/mobile-notifications.md

## Testing mobile notifications

### For users on Zulip Cloud and Zulip Server 8.0+

!!! tip ""

    Follow [these instructions](/help/view-zulip-version) to check the Zulip
    version for your organization.

To verify that mobile notifications are working as desired, you can send
yourself a test notification from the Zulip mobile app. If you belong to more
than one Zulip organization, you can separately test notifications for each
account.

{start_tabs}

{tab|mobile}

1. [Download](https://zulip.com/apps/) and install the Zulip mobile app if you
   have not done so already.

1. [Log in](/help/logging-in) to the account you want to test.

{!mobile-profile-menu.md!}

1. Tap **Settings**.

1. Tap **Notifications**.

1. Tap **Send a test notification**. If notifications are working, you will
   receive a **Test notification**.

!!! tip ""

>     If you see a banner indicating that notifications have not been set up,
>     try [logging out](/help/logging-out) of your account and [logging
>     in](/help/logging-in) again. This will help if notifications were set up
>     very recently on your Zulip server.

Reviewer: ```suggestion
    If you see a banner indicating that notifications have not been enabled
    for your Zulip server, try [logging out](/help/logging-out) of your account
    and [logging in](/help/logging-in) again. This will help if notifications
    were enabled very recently.
```

With zulip/zulip-mobile#5785 (on my plate for this week), the banner will say "enabled" rather than "set up".
